src/observable.js

```diff
@@ -114,6 +114,52 @@ extend(Observable.prototype, {
     return this._off(ANY, fn);
   },
 
+  subscribe(observer) {
+    const _this = this;
+    let closed = false;
+
+    if (typeof observer === 'function') {
+      observer = {
+        next: observer,
+        error: arguments[1],
+        complete: arguments[2]
+      };
+    }
+
+    if (observer.next) {
+      this.onValue(observer.next);
+    }
+
+    if (observer.error) {
+      this.onError(observer.error);
+    }
+
+    if (observer.end) {
+      this.onEnd(observer.complete);
+    }
+
+    return {
```

Comment:
I see. Would be cool to make it one method (O.prototype.subscribe === ESO.prototype.subscribe) and fix it in one place then.